docs/content/aws-cloudformation-hooks.md

```diff
@@ -217,7 +217,27 @@ fail (i.e. not evaluate) as soon as the property was missing, leading to the res
 and negation is a good way to work with data that might or might not be present, and results in more readable policies,
 too.
 
-## 5. Policy Enforcement Testing 
+{{< danger >}}
+Surprisingly, boolean values from CloudFormation Templates are provided to the hook in the form of **strings** (i.e.
+"true" and "false"). Policy authors must take this into account, and explicitly check for the value of these
+attributes. An example S3 bucket policy might for example want to check that public ACLs are blocked:
+
+```live:example/boolean_fail:module
```

Comment:
[nit] I think these could use read_only live blocks. But it's OK.

Reply:
Thanks! Fixed.